src/routes/trip/trip-card.js

```diff
@@ -62,7 +64,12 @@ function getLeaderData(req, tripId, userId) {
   LEFT JOIN users ON users.id = trip_members.user
   WHERE trip = ?
   ORDER BY is_owner DESC, trip_members.leader DESC, trip_members.rowid
-  `, tripId) // Display order is leaders first, followed by signup order
+  `, tripId).map(member => ({
+    ...member,
+    time_element: utils.getDatetimeElement(member.signed_up_on)
```

Comment:
although it will require you to mess with the datetime function a bit here, I think it would look better on the table if there was a `<br>` between the date and the time. Maybe just an optional `addBreak` boolean parameter.